src/static/js/modules/polyfill/array-filter-polyfill.js

```diff
@@ -0,0 +1,45 @@
+/* ==========================================================================
+ Polyfill for Array filter.
+ Copied from https://developer.mozilla.org/en-US/docs/Web
+                   /JavaScript/Reference/Global_Objects/Array/map.
+
+ Production steps of ECMA-262, Edition 5, 15.4.4.19
+ Reference: http://es5.github.io/#x15.4.4.19
+ ========================================================================== */
+
+'use strict';
+
+if ( !Array.prototype.filter ) {
+  // Ignore complexity eslint offense for polyfill.
+  Array.prototype.filter = function filter( fun ) { //eslint-disable-line
+
+    if ( typeof this === 'undefined' || this === null ) {
+      throw new TypeError();
+    }
+
+    var t = Object( this );
+    var len = t.length >>> 0;
+    if ( typeof fun !== 'function' ) {
+      throw new TypeError();
+    }
+
+    var res = [];
+    var thisArg = arguments.length >= 2 ? arguments[1] : null;
+    for ( var i = 0; i < len; i++ ) {
+      if ( i in t ) {
+        var val = t[i];
+
+        // NOTE: Technically this should Object.defineProperty at
```

Comment:
Actually, I think this okay as is. Just read strange to me, but depends how Object.defineProperty is applied.